node/flows.re

```diff
@@ -2,8 +2,6 @@ open Helpers;
 open Protocol;
 open Building_blocks;
 
-module Node = State;
```

Comment:
This is not an improvement, when we do `Node.something` I don't mean `the node state do something`, the way to read it is completely different, that's why there is this module alias here.

The only reason why the file `state.re` is not called `node.re` is because of how dune libraries naming works.

Reply:
In Tezos there are lots of module aliasing, and I find it obscures the code considerably (other Tezos developers have said this too). Therefore I believe there should not be any module aliases that are not necessary. The rule should be:
- Try to avoid conflicting names
- Only if you can't should you do a module alias.